index.xml

```diff
@@ -12005,12 +11827,51 @@ interface
                     Try running the following steps:
                     <ol>
                       <li>
+                        Let <var>O</var> be <span class='esvalue'>null</span>.
+                      </li>
+
+                      <li>
+                        If <var>target</var> is not a <a class='dfnref'
+                        href='#dfn-namespace'>namespace</a>, and <var>op</var> is not a <a
+                        class='dfnref' href='#dfn-static-operation'>static operation</a>:
+
+                        <ol>
+                          <li>
+                            If the <span class='esvalue'>this</span> value is <span
+                            class='esvalue'>null</span> or <span class='esvalue'>undefined</span>,
+                            set <var>O</var> to <var>realm</var>'s <a class='dfnref external'
+                            href='https://html.spec.whatwg.org/multipage/webappapis.html#concept-realm-global'>global
+                            object</a>. (This will subsequently cause a <span
```

Comment:
There's no way `target` can be the appropriate global object.  `target` isn't an object at all.

What you probable meant is "if the global does not implement `target`" or something like that?